include/spirv-tools/optimizer.hpp

```diff
@@ -49,7 +50,7 @@ class Optimizer {
     // below; for out-of-tree passes, use this constructor instead.
     // Note that this API isn't guaranteed to be stable and may change without
     // preserving source or binary compatibility in the future.
-    PassToken(std::unique_ptr<opt::Pass>&& pass);
+    PassToken(std::unique_ptr<opt::PassToken>&& pass);
```

Comment:
This is public facing change. But, it changes from one opaque type to a second, so I think it's safe.